rolemanager/module.py

```diff
@@ -0,0 +1,120 @@
+from typing import List, Set
+
+import discord
+from discord.ext import commands
+
+from pie import utils, check, i18n
+from pie.utils.objects import ConfirmView, ScrollableEmbed
+
+_ = i18n.Translator("modules/fsi").translate
+
+
+class RoleManager(commands.Cog):
+    def __init__(self, bot: commands.Bot):
+        self.bot = bot
+
+    # HELPER FUNCTIONS
+    def _create_embeds(ctx, title: str, description: str) -> List[discord.Embed]:
+        elements = []
+        """Create embed for member list.
+        Args:
+            ctx: Command context.
+            option: Item's title.
+            description: list of items.
+        Returns: :class:`discord.Embed` information embed
+        """
+        chunk_size = 15
+
+        for i in range(0, len(description), chunk_size):
+
+            page = utils.discord.create_embed(
+                author=ctx.author,
+                title=title,
+                description="\n".join(description[i : i + chunk_size]),
+            )
+
+            elements.append(page)
+
+        return elements
+
+    def _get_intersection(role_base, role_remove) -> Set[discord.Member]:
+        base_members = set(role_base.members)
+        remove_members = set(role_remove.members)
+
+        role_intersection = base_members & remove_members
+        return role_intersection
+
+    # MAIN
+    @commands.guild_only()
+    @commands.group(name="rolemanager")
+    @check.acl2(check.ACLevel.MOD)
+    async def rolemanager_(self, ctx):
+        """
+        Preview and remove selected roles from members with specified based role.
+        """
+        await utils.discord.send_help(ctx)
+
+    @check.acl2(check.ACLevel.MOD)
+    @rolemanager_.command(name="preview")
+    async def rolemanager_preview(
+        self, ctx, role_base: discord.Role, role_remove: discord.Role
+    ):
+        """
+        List users to remove selected role from members with base role.
+        """
+        members_list = RoleManager._get_intersection(role_base, role_remove)
+
+        if members_list:
+
+            title = _(ctx, "Members with forbidden role")
+
+            member_list = list(str(member) for member in members_list)
```

Comment:
It's not good practice to assign two values of different type into same variable. I'd suggest to change this variable's name to `name_list` or something like this.

Reply:
Changed name to name_list.